src/guides/v2.4/release-notes/commerce-2-4-2.md

```diff
@@ -0,0 +1,1455 @@
+
+To support a complete workflow for accepting online payments, this feature also:
+
+*  Overrides existing payment method templates during initial checkout to maintain PCI compliance.
+
+*  Maintains compatibility with third-party, custom, and community-developed payment methods.
+
+*  Notifies buyers by email when they need to add payment details to an approved purchase order.
+
+*  Enables customization of emails from the Admin, which allows merchants to use templates that are consistent with their branding.
+
+*  Introduces an `Approved – Pending Payment` state for purchase orders to clearly show when action is required.
+
+*  Prevents discount codes from being added or removed at the final payment step, which ensures that the order total amount remains unchanged for approved purchase orders.
+
+*  Allows buyers to change the payment method during the final payment step to maintain flexibility and increase conversion.
+
+This release also includes multiple bug fixes. See [B2B Release Notes]({{page.baseurl}}/release-notes/b2b-release-notes.html).
+
+### Media Gallery
+
+**New Role Resources for Media Gallery**. This release provides merchants the ability to limit administrator access to only the Media gallery and to control who can perform these actions:
+
+*  Insert media assets into content
+
+*  Upload assets
+
+*  Edit assets details
+
+*  Delete assets from the Media Gallery
+
+*  Manage folder structure.
+
+**Web-optimized images in content**. Merchants can now use [web-optimized image rendition](https://docs-beta.magento.com/user-guide/cms/media-gallery-image-optimization.html) in content instead of high resolution images. The original image remains unmodified in the Media Gallery, and the image rendition is dynamically generated when the image is inserted in the content.
+
+### AWS S3 support enhancements
+
+Amazon Simple Storage Service (AWS S3) support has been enhanced to include support for:
+
+*  Object storage and future extensibility
+
+*  [Storing media files]((https://docs-beta.magento.com/user-guide/system/data-import-product-images.html#method-3-import-images-with-remote-storage) on AWS S3
+
+Support for AWS S3 has been added to all modules including B2B, PageBuilder, and Adobe Stock Integration.
+
+### Magento Functional Testing Framework (MFTF)
+
+MFTF 3.2.0 is now available. This release introduces error tolerance in both tests and test suit generation.  Additional enhancements and bug fixes are described in the [Magento Functional Testing Framework Changelog](https://github.com/magento/magento2-functional-testing-framework/blob/develop/CHANGELOG.md).
```

Comment:
Should this be 3.2.1? See https://github.com/magento/magento2-functional-testing-framework/blob/3.2.1/CHANGELOG.md#321.

Reply:
thanks for catching that!  Changed in both files.